jbmc/src/java_bytecode/remove_instanceof.cpp

```diff
@@ -39,7 +40,7 @@ class remove_instanceoft
   namespacet ns;
   class_hierarchyt class_hierarchy;
 
-  std::size_t lower_instanceof(
+  bool lower_instanceof(
```

Comment:
This refactoring should probably be in a separate commit.

Reply:
This isn't refactoring, but rather removing a return value that is no longer needed due to the remaining changes